website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -113,6 +129,11 @@ To create a policy set:
 1. Choose a policy set scope:
    - **Policies enforced globally:** Terraform Cloud automatically enforces this global policy set on all of an organization's existing and future workspaces.
    - **Policies enforced on selected projects and workspaces:** Use the text fields to find and select the workspaces and projects to enforce this policy set on. This affects all current and future workspaces for any chosen projects.
+
+1. **(Sentinel Only)** Choose a policy set type:
+   - **Standard:** This is the default workflow. Sentinel policy set is run as a policy check in Terraform Cloud.
+   - **Enhanced:** Sentinel policy set is run as a policy evaluation in Terraform Cloud.
```

Comment:
Suggestion: Maybe hyperlink `policy check` and `policy evaluation` if needed? maybe not

```suggestion
   - **Standard:** This is the default workflow. A Sentinel policy set uses a policy check in Terraform Cloud.
   - **Enhanced:** A Sentinel policy set uses a policy evaluation in Terraform Cloud.
```